### PR TITLE
FIX selectable to be really not interactive

### PR DESCRIPTION
**Pull request by jasmainak**
cc @teonbrooks 

selectable was not stopping interactivity which broke real-time examples ...

### CODE REVIEW

File: mne/realtime/fieldtrip_client.py [hunk 1]
```diff
@@ -185,7 +185,7 @@ def _guess_measurement_info(self):
                 this_info['cal'] = 1.0
 
                 this_info['ch_name'] = ch
-                this_info['loc'] = None
+                this_info['loc'] = np.zeros(9)
```
Comment (jasmainak) [inline mne/realtime/fieldtrip_client.py]: Now the viz functions assume that loc is 0s if it is unknown ...

Comment (Eric89GXL) [inline mne/realtime/fieldtrip_client.py]: I think it's 12, not 9

Comment (teonbrooks) [inline mne/realtime/fieldtrip_client.py]: ok

**Comment by teonbrooks** [comment]
now LGTM :)

**Comment by jaeilepp** [comment]
Thanks @jasmainak

**Pull request merged**